doc/guides/west/repo-tool.rst

```diff
@@ -32,16 +32,16 @@ like this:
 
 .. code-block:: none
 
-   zephyrproject
-   ├── .west
+   zephyrproject/                 # west topdir
+   ├── .west/
    │   └── config
-   ├── zephyr
-   │   ├── west.yml
+   ├── zephyr/                    # .git/ ignored by west
```

Comment:
We don't guarantee that west won't ever read from the .git repository. We just guarantee west won't write to this repository at this time.

Reply:
OK, so how about `# .git/ never modified/touched/changed by west`?

Reply:
> how about `# .git/ never modified/touched/changed by west`?

Sure, pick your favorite though, all those slashes make me :(